tests/test_builder.py

```diff
@@ -438,6 +438,41 @@ def test_epr_keep_info():
         [
             GenericInstr.CREATE_EPR,
             GenericInstr.WAIT_ALL,
+            PatternWildcard.ANY_ZERO_OR_MORE,
+            GenericInstr.LOAD,
+            GenericInstr.BLT,
+            GenericInstr.SET,
+            GenericInstr.QALLOC,
+            GenericInstr.INIT,
+            PatternWildcard.BRANCH_LABEL,
+        ]
+    )
+
+
+def test_epr_recv_keep_info():
+    DebugConnection.node_ids = {
+        "Alice": 0,
+        "Bob": 1,
+    }
+
+    epr_socket = EPRSocket("Bob")
+
+    with DebugConnection("Alice", epr_sockets=[epr_socket]) as conn:
+        eprs, infos = epr_socket.recv_keep_with_info()
+        epr, info = eprs[0], infos[0]
+
+        _ = epr.measure(store_array=False)
+        with info.generation_duration.if_ge(1337):
```

Comment:
Why 1337?

Reply:
No particular reason; any number had to be used (it only checks for the instruction type, not the operand; that functionality is not implemented yet). This number was already used in older tests as well.